client/dashboard/sites/settings-sftp-ssh/summary.tsx

```diff
@@ -0,0 +1,52 @@
+import { useQuery } from '@tanstack/react-query';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { file } from '@wordpress/icons';
+import { siteSftpUsersQuery, siteSshAccessStatusQuery } from '../../app/queries';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { canUseSftp, canUseSsh } from '../../utils/site-features';
+import type { Site } from '../../data/types';
+import type { Density } from '@automattic/components/src/summary-button/types';
+
+export default function SftpSshSettingsSummary( {
+	site,
+	density,
+}: {
+	site: Site;
+	density: Density;
+} ) {
+	const { data: sftpUsers } = useQuery( {
+		...siteSftpUsersQuery( site.slug ),
+		enabled: canUseSftp( site ),
+	} );
+
+	const { data: sshAccessStatus } = useQuery( {
+		...siteSshAccessStatusQuery( site.slug ),
+		enabled: canUseSsh( site ),
+	} );
+
+	const sftpEnabled = sftpUsers && sftpUsers.length > 0;
+
+	const sshEnabled = sshAccessStatus?.setting === 'ssh';
+
+	const badges = [
+		{
+			text: sftpEnabled ? __( 'SFTP enabled' ) : __( 'SFTP disabled' ),
```

Comment:
I don't think badges for disabled states are strictly needed in the settings list. They can add a lot of unnecessary clutter.

<img width="629" alt="Screenshot 2025-06-03 at 11 41 51" src="https://github.com/user-attachments/assets/d24a628b-ca3b-43c9-aef6-19de03e81a6f" />

This feels like something we should agree on at the design system-level though. @jasmussen, @jameskoster, do you have an opinion on whether to display badges for disabled states in SummaryButtons?

Reply:
Okay, happy to leave it up to the specific context. 👍